.github/ISSUE_TEMPLATE/bug_template.yaml

```diff
@@ -0,0 +1,40 @@
+name: Bug Report
+description: File a bug report
+title: "[Bug]: "
+labels: [bug]
+body:
+ - type: markdown
+   attributes:
+    value: |
+     Thanks for taking the time to fill out this bug report!
+ - type: textarea
+   id: what-happened
+   attributes:
+    label: What happened?
```

Comment:
nit: Can we say `Describe the bug` instead of `What happened`?

Reply:
+1